src/website/app/demos/Radio/examples/radio/controlled.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import { Component } from 'react';
+import DemoForm from '../../components/DemoForm';
+import Radio from '../../../../../../Radio';
+
+export default {
+  id: 'controlled',
+  title: 'Controlled',
+  description: `In a controlled Radio, the checked state is handled by a React
+component. Set the checked state with the \`checked\` prop and provide an
+\`onChange\` handler.`,
+  scope: { Component, DemoForm, Radio },
+  source: `
+  () => {
+    class MyForm extends Component {
+      constructor(props) {
+        super(props);
+
+        this.state = {
+          value: 'quartz'
```

Comment:
when I use arrow keys to select the other one, and then TAB off, this resets and looks like it's broken b/c the live editor is rerunning the constructor. not sure if there's a good way to fix that...

Reply:
That's really weird.  React-live runs the constructor when you tab into it.  I think we'll have to let this one go for the time being.